map/styled_point.cpp

```diff
@@ -0,0 +1,54 @@
+#include "map/styled_point.hpp"
+
+#include "base/stl_add.hpp"
+#include "base/string_utils.hpp"
+
+#include "std/fstream.hpp"
+#include "std/algorithm.hpp"
+#include "std/auto_ptr.hpp"
+
+
+graphics::DisplayList * StyledPoint::GetDisplayList(UserMarkDLCache * cache) const
+{
+  return cache->FindUserMark(UserMarkDLCache::Key(GetStyle(), graphics::EPosAbove, GetContainer()->GetDepth()));
```

Comment:
If you change default API behavior to draw bookmarks instead of violet API points, it will break UI in already existing apps.
It should use “default” old style if this parameter (“s”) is omitted from the API call.